README.md

```diff
@@ -65,29 +66,33 @@ The following provides instructions for how to build docs locally for visualizat
 4. Install MyST Parser `pip install myst_parser`
 5. CD into the docs directory `cd [path to this repo]/docs`
 6. Run `make html`
-7. This will generate HTML in docs/build/html. Setup a web server with the web root as docs/build/html or open docs/build/html/index.html in a browser.
- 
+7. This will generate HTML in `docs/build/html`. Setup a web server with the web root as `docs/build/html` or open `docs/build/html/index.html` in a browser.
```

Comment:
I would add something here about using DDEV - you can just run 'ddev config' and state docs/build/html as the root, then 'ddev start' and it'll run a web server for you. it's how we work with mautic/mautic so most folks are used to it.

Reply:
I haven't run/done this myself, so I don't want to assume. 😅
But I have a question for clarification.

Do you want to add a step using DDEV or do you want to change the step here with using DDEV?

Also, can you please point me to docs or instructions about DDEV?

Thank you.